src/templates/items/parts/item-descriptors.html

```diff
@@ -0,0 +1,9 @@
+            {{!-- Descriptors Template --}}
+            <div class="form-group stacked weapon-properties">
```

Comment:
It makes more sense semantically to use a custom class here instead of reusing `weapon-properties`. Maybe something like `item-descriptors`. Then in `src/less/item.less`, you can update this rule like this:

```less
/* ----------------------------------------- */
/*  Item Actions                             */
/* ----------------------------------------- */
.item-descriptors label.checkbox,
.weapon-properties label.checkbox {
  flex: 0 0 130px;
}
```